distribution/ddf-common/src/main/resources-filtered/etc/custom.system.properties

```diff
@@ -95,6 +95,8 @@ solr.username=admin
 # Comma-separated list of metacard types that do not support optimistic updates.
 solr.commit.nrt.metacardTypes=workspace,metacard.query,metacard.list,query-template,attribute-group,resource-note
 
+migration.supported.versions = 2.13.9
```

Comment:
Note: This list will most likely be expanded before merging this PR, based on what versions we decide and what we test out.

Reply:
Yes I've added a comment of that nature in the new `migration.properties` file.